translation-web/src/main/java/eu/europeana/api/translation/web/TranslationController.java

```diff
@@ -63,5 +76,45 @@ private void validateRequest(TranslationRequest translationRequest) throws Param
       throw new ParamValidationException(null, null, ERROR_INVALID_PARAM_VALUE, new String[] {LanguagePair.generateKey(TranslationAppConstants.SOURCE_LANG, TranslationAppConstants.TARGET_LANG) , languagePair.toString()});
     }
   }
+  
+  private TranslationResponse getCombinedCachedAndTranslatedResults(TranslationRequest translRequest) throws EuropeanaI18nApiException {
```

Comment:
this is to be moved in the TranslationWebService and called within the translate method